en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -1,7 +1,7 @@
 # Change the Default Mediation Flow of API Requests
```

Comment:
Can we structure this document into sections? for e.g. the section to do on publisher /developer  and have shorter sub sections?